index.js

```diff
@@ -248,13 +217,20 @@ module.exports = {
   run,
   runScan,
   installGrype,
-  mergeResults,
-  findContent,
-  loadContent,
 };
 
 if (require.main === module) {
-  run().catch((err) => {
-    throw new Error(err);
-  });
+  const entrypoint = core.getInput("run");
```

Comment:
I decided to use a parameter to switch functionality here instead of compiling multiple different files; there are some limitations to the `ncc` tool used to build the actual `dist/index.js` and limitations in github actions that mean you either need to do this or compile multiple distinct scripts, all of which have duplicated code. I did the latter approach in `sbom-action` but it's pretty annoying and I'm probably going to swap to this method to simplify things and speed up the commit/build cycle.